_security-analytics/threat-intelligence/api/monitor.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: default
+title: Monitor APIs
+parent: Threat intelligence APIs
+grand_parent: Threat intelligence
+nav_order: 35
+---
+
+# Monitor APIs
+
+The threat intelligence Monitor API creates, searches, and updates monitors using the monitor's ID.
```

Comment:
Can we say what the monitor API does or why we use it?

Reply:
I'll link to the monitor documentation.